package.json

```
@@ -7,21 +7,36 @@
   "private": true,
   "scripts": {
     "build": "webpack --mode production",
-    "start": "webpack serve --mode development"
+    "start": "webpack serve --mode development",
+    "lint": "eslint --fix .",
```

It looks like this script doesn't lint jsx files. I believe you need to specify the extension:
```suggestion
    "lint": "eslint --ext .jsx,.js --fix .",
```